EpiAware/src/inference-methods.jl

```diff
@@ -0,0 +1,51 @@
+"""
+
+Do inference on an EpiAware model.
+"""
+function _epi_aware(y_t,
+        time_steps;
+        epi_model::AbstractEpiModel,
+        latent_model::AbstractLatentModel,
+        observation_model::AbstractObservationModel,
+        nsamples,
+        nchains,
+        pf_ndraws = 10,
+        pf_nruns = 10,
+        fixed_parameters = (;),
+        pos_shift = 1e-6,
+        executor = Transducers.ThreadedEx(),
+        adtype = AutoReverseDiff(true),
+        maxiters = 10,
+        kwargs...)
+    gen_mdl = make_epi_aware(missing, time_steps; epi_model,
```

Comment:
rather than all this repeating (which is dull and dangerous) can make this a small function?

Reply:
Yeah I ditched this for now